src/components/Navbar.tsx

```diff
@@ -18,48 +18,40 @@ import { useDispatch, useSelector } from "react-redux";
 import { RootState } from "@/state/store";
 import { useUser } from "@auth0/nextjs-auth0/client";
 import { login, logout } from "@/state/user/authSlice";
-import { useEffect } from "react";
+import { useEffect, useState } from "react";
 import { toast } from "react-toastify";
 import { adminNavItems } from "./NavItems";
 import { initialNavItems } from "./NavItems";
-import useDataListener from "@/lib/firebase/dataListener";
 import log from "loglevel";
+import { showToast } from "@/helpers/toast";
+import { TOAST_TYPES } from "@/constants/toastEnums";
 
 interface Props {
   /**
    * Injected by the documentation to work in an iframe.
    * You won't need it on your project.
    */
-  window?: () => Window;
+  windows?: () => Window;
```

Comment:
@LaithAbusada what do you mean by `windows` here? Why do you need a prop that returns the `Window` object?

Reply:
i changed the variable's name, as i needed to use this statement to change the navbar color from transparent to solid when a user scrolls :

 useEffect(() => {
    window.addEventListener("scroll", () => {
      if (window.scrollY > 100) {
        setColor("#5d13cb");
      } else {
        setColor("transparent");
      }
    });
  }, []);

but the MUI appbar was already using a variable called window, but i needed to use the window to adjust the color, so i just renamed the variable to windows